gemrb/core/Inventory.cpp

```diff
@@ -1468,7 +1468,8 @@ void Inventory::BreakItemSlot(ieDword slot)
 	const Item *itm = GetItemPointer(slot, Slot);
 	if (!itm) return;
 	//if it is the magic weapon slot, don't break it, just remove it, because it couldn't be removed
-	if(slot ==(unsigned int) SLOT_MAGIC) {
+	//or for pst, just remove it as there is no breaking (the replacement item is a sound)
+	if(slot ==(unsigned int) SLOT_MAGIC || core->HasFeature(GF_HAS_PICK_SOUND)) {
```

Comment:
This is the correct fix, nice job. :)
Since you're modifying the line, please also fix the inherited stylistic issues:
```suggestion
	if (slot == (unsigned int) SLOT_MAGIC || core->HasFeature(GF_HAS_PICK_SOUND)) {
```